app/js/track_events.js

```diff
@@ -0,0 +1,42 @@
+$(function() {
+  'use strict';
+
+  trackMenuClick();
+
+  analytics.trackForm($('.Nav form'), 'Subscribed Newsletter in the menu', getFormProperties);
+  analytics.trackForm($('.Newsletter form'), 'Subscribed Newsletter in the main content', getFormProperties);
+
+  $('[data-track-link]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLink'));
+  });
+
+  $('[data-track-link-mailto]').each(function() {
+    var $element = $(this);
+    analytics.trackLink($element, $element.data('trackLinkMailto'), getMailFromMailTo);
+  });
+
+  function trackMenuClick() {
+    $('#BurgerBtn').on('click', function(event) {
+      var $btn = $(event.target);
+      if ($btn.hasClass('BurgerBtn--open')) {
+        analytics.track('Opened main menu');
+      } else {
+        analytics.track('Closed main menu');
+      }
```

Comment:
@azevedo-252 I just noticed, it must have been a typo, because I thought I had said "2. sounds good to me"

Still, I don't have a problem with 1 either, especially if it's already done